### ISSUE TITLE
minmax should allow fr

### ISSUE THREAD

**Issue by lukashambsch** [SentiaAnalytics/bs-css#118]
The minmax attribute should let you use fr.

I'm currently getting this error:

```
gridTemplateColumns([
  13 ┆   `repeat((`autoFit, `minmax((px(150), fr(1.))))),
  14 ┆ ]),
  15 ┆ gridGap(px(12)),

  This has type:
    [> `fr(float) ]
  But somewhere wanted:
    Css.length
  The second variant type does not allow tag(s) `fr
```

**Comment by JakubMarkiewicz** [comment]
I can take this

**Comment by giraud** [comment]
Fixed in 8.0.3
